packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -558,7 +558,7 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
       // It isn't safe to trigger the ScrollMetricsNotification if we are in
       // the middle of rendering the frame, the developer is likely to schedule
       // a new frame(build scheduled during frame is illegal).
-      if (_lastMetrics != null && !_haveScheduledUpdateNotification) {
+      if (!_haveScheduledUpdateNotification) {
```

Comment:
@xu-baolin @Piinks 
any reason we don't want to send when it initialize for the first time?

My use case here is the render object listens to the scrollable metrics before it lays out the first time, it wants to be notified when the scrollable metrics initialize.

Reply:
If this doesn't make sense, we probably need a hook, something like didInitializeScrollMetrics

Reply:
There is no special reason, it's just that we didn't find the necessary scene at the time. If you need it, I think we can do it.

Reply:
I post #87421 to land this change.